tally

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+
+import json
+import argparse
+import os
+
+import pyrankvote
+from pyrankvote import Candidate, Ballot
```

Comment:
This should have a project spec so we can define and lock the dependencies.

Also, I'd like a comment linking to the TOF charter where it's defined that we use ranked-choice, as well as a link to the pyrankvote documentation.